Automattic/jetpack
# [Package] Constants: fix tests

#### Changes proposed in this Pull Request:

Trying to fix the following tests:

```
Error:

1) Test_Constants::test_jetpack_constants_get_constant_null_when_not_set
Mockery\Exception\InvalidCountException: Method apply_filters('jetpack_constant_default_value', NULL, 'UNDEFINED') from Mockery_0 should be called exactly 1 times but called 0 times.
test-constants.php:35

Failure:

1) Test_Constants::test_jetpack_constants_get_constant_use_filter_value
Failed asserting that null matches expected 'test value'.
test-constants.php:153
```

I've simply adjusted the `once()` call on the expectation to after the `with()`, no clue why that should make a difference from [what I see in docs](https://brain-wp.github.io/BrainMonkey/docs/functions-expect.html) 🤷 

If adjusted to `twice()` with the new ordering, I can see the tests indeed then fail and run just once (while expecting to have run twice) - so the tests appear to be working?

#### Jetpack product discussion

Internal: p1629224704045800-slack-CBG1CP4EN

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Pull down changes, and open the `projects/packages/constants` dir in your terminal, then run `composer phpunit --verbose` to run the updated tests.
* Verify that all tests are passing.
